initialise/MirahezeFunctions.php

```diff
@@ -1249,4 +1252,37 @@ public static function onMediaWikiServices() {
 			unset( $GLOBALS['globals'] );
 		}
 	}
+
+	/**
+	 * @param AuthenticationResponse $response
+	 * @param User | null $user
+	 * @param string | null $username
+	 */
+	public static function onAuthManagerLoginAuthenticateAudit( $response, $user, $username ) {
```

Comment:
type hints (including return value of void) please

Reply:
Also add $extraData param even if it's unused just to resemble the actual hook interface.